minui/graphics_fbdev.c

```diff
@@ -300,3 +311,57 @@ fbdev_exit(minui_backend *backend UNUSED)
 
 	gr_draw = NULL;
 }
+
+/* ------------------------------------------------------------------------ */
+
+static void *save_buf[2] = { NULL, NULL };
+
+static void
+fbdev_save(minui_backend *backend UNUSED)
+{
+	if (!(save_buf[0] = malloc(gr_draw->height * gr_draw->row_bytes))) {
```

Comment:
If fbdev_save() is called multiple times without calling fbdev_restore() in between, the already allocated
buffers are leaked.